update load interited widget code lab (#94)
flutter/codelabs: dartpad_codelabs/src/inherited_widget/step_01/instructions.md

# Define the State Data Structure

> The first thing to do is defining a data structure to store your app state.

Reviewer: ```suggestion
The first thing to do is define a data structure to store your app state.
```

> To do that, we need to find what should be considered as the app state.

Reviewer: ```suggestion
To do that, we need to determine what should be considered the app state.
```

> We know there are at least two components requires state, the product list

Reviewer: ```suggestion
We know there are at least two components require state, the product list
```
